### DISCUSSION TITLE
[Feature Request] Look at Stadia's own FPS

### DISCUSSION THREAD

**Discussion by wodnerpie** [ChristopherKlay/StadiaEnhanced#227] [Ideas & Requests]
First of all I understand that this, may be possibly impossible, but I think it would be really really great to implement such a nice feature
What I mean is, for example when I'm playing Dirt 5 and I see (by myself) that there is a FPS loss, I can never know if it's my PC having issues decoding VP9, if it's my Internet that is playing with me or it's Stadia that can't keep up with the game's graphics
The idea would be to, alongside the current FPS monitor (that monitors the local FPS, not the ones that Stadia processes), to add a "real" FPS tab, that would show how much FPS Stadia actually sends

**Comment by ChristopherKlay** [comment]
While i would love to implement this, it currently isn't possible.

Google would either need to provide such performance data directly, or use it in some form (i.e. a diagnostic / statistics page) where it could be accessed, since none of the game specific data is shared via for example the WebRTC session (where the stream FPS come from).
